daft/expressions/expressions.py

```diff
@@ -854,6 +854,18 @@ def capitalize(self) -> Expression:
         """
         return Expression._from_pyexpr(self._expr.utf8_capitalize())
 
+    def left(self, pattern: int | Expression) -> Expression:
```

Comment:
Just one last nit here, could you rename this (and all the other occurrences of `pattern`) to `nchars` like you did for the tests? Just for consistency 😄 , after this it should be good to merge!